cookbook/logging/monolog.rst

```diff
@@ -220,7 +220,7 @@ easily. Your formatter must implement
 Adding some extra Data in the Log Messages
 ------------------------------------------
 
-Monolog allows to process the record before logging it to add some
+Monolog allows you to process the record before logging it to add some
```

Comment:
Here's a quick discussion on the phrase "allows to":

http://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english